src/main/java/org/apache/commons/imaging/ImageParser.java

```diff
@@ -563,27 +561,22 @@ public final BufferedImage getBufferedImage(final File file, final Map<String, O
      * <p>The params argument provides a mechanism for individual
      * implementations to pass optional information into the parser.
      * Not all formats will support this capability.  Currently,
-     * some of the parsers do not check for null arguments. So in cases
-     * where no optional specifications are supported, application
-     * code should pass in an empty instance of an implementation of
-     * the map interface (i.e. an empty HashMap).
+     * some of the parsers do not check for null arguments.</p>
```

Comment:
Should there be a future issue to make sure that all parsers check for null?

Reply:
Good idea! Will create it now.

Reply:
Done! https://issues.apache.org/jira/browse/IMAGING-323